WalletWasabi.Fluent/ViewModels/Dialogs/CancelTransactionDialogViewModel.cs

```diff
@@ -0,0 +1,81 @@
+using System.Linq;
+using System.Reactive;
+using System.Threading.Tasks;
+using NBitcoin;
+using ReactiveUI;
+using WalletWasabi.Blockchain.TransactionBuilding;
+using WalletWasabi.Blockchain.Transactions;
+using WalletWasabi.Fluent.Extensions;
+using WalletWasabi.Fluent.ViewModels.Dialogs.Base;
+using WalletWasabi.Logging;
+using WalletWasabi.Wallets;
+
+namespace WalletWasabi.Fluent.ViewModels.Dialogs;
+
+[NavigationMetaData(Title = "Cancel Transaction")]
+public partial class CancelTransactionDialogViewModel : DialogViewModelBase<Unit>
```

Comment:
Return type `Unit` is a sign of it should be `DialogViewModelBase` because it doesn't have to be awaitable. Use `RoutableViewModel`